src/sssom/util.py

```diff
@@ -504,17 +504,38 @@ def compare_dataframes(df1: pd.DataFrame, df2: pd.DataFrame) -> MappingSetDiff:
     return d
 
 
-def dataframe_to_ptable(df: pd.DataFrame, *, inverse_factor: float = None):
+def add_default_confidence(df: pd.DataFrame) -> pd.DataFrame:
+    """Add `confidence` column to DataFrame if absent and initializes to 0.95.
+
+    If `confidence` column already exists, only fill in the None ones by 0.95.
+
+    :param df: DataFrame whose `confidence` column needs to be filled.
+    :return: DataFrame with a complete `confidence` column.
+    """
+    df[CONFIDENCE] = df.get(CONFIDENCE, 0.95)
+    df.loc[df[CONFIDENCE].isnull(), CONFIDENCE] = 0.95
+    return df
+
+
+def dataframe_to_ptable(
+    df: pd.DataFrame, *, inverse_factor: float = None, default_confidence: bool = False
```

Comment:
Rename to just "confidence"?

Reply:
I'll rename to `default_confidence` since there's already a variable named `confidence` down below in the same function.